cluster-autoscaler/cloudprovider/clusterapi/clusterapi_converters_test.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clusterapi
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	apiVersion = "machine.openshift.io/v1beta1"
+	kind       = "MachineSet"
+	name       = "Foo"
+	namespace  = "Bar"
+)
+
+func TestConverterNewMachineSetFromUnstructured(t *testing.T) {
```

Comment:
Will there be tests for the other converters? Could maybe follow up with those 🤔 

One interesting test to add on top of this as well may be to create a complete {Machine, MachineSet, MachineDeployment} and then convert it to unstructured and back again using the converters in this package and check the conversion doesn't lose any data?